examples/pvc-with-local-volume/pvc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: local-volume-pvc
+  annotations:
+    volumeType: local
```

Comment:
We could rename this variable, do we want something like `rancher.io/local-path-provisioner/volume-type`

Reply:
I think this is fine. Any concern?

Reply:
Not a big concern, just thinking that if we lock down the annotation key to use a rancher prefix, it is 100% tied to this provisioner.

Could be some weird edge case where a person uses `volumeType` annotation as some sort of internal value for tracking, not knowing that it changes pv creation behavior.

Reply:
@derekbit If we are fine leaving this as-is, I can close this comment and merge?